src/gui/tray/Window.qml

```diff
@@ -383,13 +383,21 @@ Window {
                                 font.bold: true
                             }
                             Label {
-                                id: currentAccountServer
+                                id: currentAccountStatus
                                 width: Style.currentAccountLabelWidth
-                                text: UserModel.currentUser.server
+                                text: UserModel.currentUser.status
                                 elide: Text.ElideRight
                                 color: "white"
                                 font.pixelSize: Style.subLinePixelSize
                             }
+//                            Label {

```

Comment:
I guess this can go? Or maybe not... What's the design we intend there?

Reply:
atm the intention is to think about it...

Reply:
OK. My understanding is that with the removal of that lavel and the introduction of the other one the server name would disappear and we'd have the status string being displayed instead. I wonder then if that wouldn't lead to user complaints if they have the same account name on several servers, they wouldn't know anymore which one is the one being currently selected.

@jancborchardt? Any opinion?

Reply:
Yeah, it's difficult there. The basic 2 options would be:
- We just remove the server name there so we only have 2 lines, name and status in subline. Server name would still be in the dropdown menu.
- We have 3 lines: name, status and server. This is very crowded though.

Some possible enhancements and inbetweeners:
- If possible, we also change the header color (currently alwqys blie) to the themed primary color of the selected account's instance. Just like on Android basically. Is this possible?
- If we go for the 3 lines, we only need to do that in the header when there is more than 1 account is configured anyway. If you only have 1 account, having that info in the dropdown only is absolutely fine.

Reply:
I'm wondering, don't we have a third option which would be:
 * We have two lines: name@server, status

Not sure if that's a good option.

As for changing the header color I guess this is trivial if I understand which color we're talking about. I guess that's a different one than the one from the default avatar? In which case I guess the avatar would be enough to disambiguate...

Reply:
> name@server, status

Right, so how about this for the header:
- When only one account is connected, we show: **name, status**. Cause server is not important as it’s always the same. Also it’s shown in the 3-line entry in the dropdown menu)
- When multiple accounts are connected, we show: **server, status**. Cause name@server is a bit nerdy and can be confused for email, also is a bit long so likely not being shown in full, and the name is less important than the server for distinction?

What do you think @DominiqueFuchs since we worked on that together. :)

> As for changing the header color I guess this is trivial if I understand which color we're talking about. I guess that's a different one than the one from the default avatar?

Cool! Yes it’s a different one, it would be the Nextcloud blue, or whatever the server set in theming. cc @juliushaertl @tobiasKaminsky on how to get the primary color / primary theming color via API. :)

Reply:
> > name@server, status
> 
> Right, so how about this for the header:
> 
>     * When only one account is connected, we show: **name, status**. Cause server is not important as it’s always the same. Also it’s shown in the 3-line entry in the dropdown menu)
> 
>     * When multiple accounts are connected, we show: **server, status**. Cause name@server is a bit nerdy and can be confused for email, also is a bit long so likely not being shown in full, and the name is less important than the server for distinction?

And since we got the avatar anyway... Sounds like a good compromise to me. 

> What do you think @DominiqueFuchs since we worked on that together. :)
> 
> > As for changing the header color I guess this is trivial if I understand which color we're talking about. I guess that's a different one than the one from the default avatar?
> 
> Cool! Yes it’s a different one, it would be the Nextcloud blue, or whatever the server set in theming. cc @juliushaertl @tobiasKaminsky on how to get the primary color / primary theming color via API. :)

But then don't we get a high chance of collision again? Most of the servers I use don't override that color for instance.